node/node_test.go

```diff
@@ -56,6 +57,23 @@ func TestNewNode(t *testing.T) {
 		P2PBootPeers:        "",
 	}
 
-	_, err := node.New(config, "v0.3")
+	n, err := node.New(config, "v0.3")
 	require.NoError(t, err)
+
+	ctx, cancel := context.WithCancel(context.Background())
+	cancel()
+
```

Comment:
Are the changes to this file related to the PR?

Reply:
it improves coverage of `node.go: lines 237-241` (before merge 221-225 as asked)

Reply:
Sure